pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: args
-version: 2.3.1
+version: 2.3.2-dev
 description: >-
  Library for defining parsers for parsing raw command-line arguments into a set
  of options and values using GNU and POSIX style options.
 repository: https://github.com/dart-lang/args
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.18.0 <3.0.0'
```

Comment:
Doesn't `package:lints` just need 2.17.0`?

Reply:
🤷 – the universe has moved on to 2.18 – don't see much reason to keep this minimal.